project/plugins.sbt

```diff
@@ -0,0 +1,6 @@
+resolvers += "bintray-spark-packages" at "https://dl.bintray.com/spark-packages/maven/"
+resolvers += Resolver.bintrayIvyRepo("s22s", "sbt-plugins")
+
+addSbtPlugin("org.spark-packages" % "sbt-spark-package" % "0.2.7-astraea.1")
+// addSbtPlugin("org.spark-packages" % "sbt-spark-package" % "0.2.6")
```

Comment:
I'm using a modified sbt-spark-package because data bricks hasn't updated the normal one to support SBT 1.x (the latest and greatest of SBT).

see: https://github.com/databricks/sbt-spark-package/pull/41